docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+          <listitem>
+            <para>Modify the admin.tfvars file, once again add your user name
+            and your email.</para>
+          </listitem>
+
+          <listitem>
+            <para>If you are using the Azure portal, find the setting for
+            auto_launch_eclwatch and set it to false as follows:</para>
+
+            <programlisting>auto_launch_eclwatch = false</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Save the File</para>
+          </listitem>
+        </orderedlist>
+      </sect3>
+
+      <sect3 id="InitializeTheTerraformModles">
+        <title>Initializing the Terraform Modules</title>
+
+        <para>After configuring the modules, the next step is to initialize.
+        The <emphasis>Terraform init</emphasis> command declares the current
+        working directory as the root or the calling module. During this
+        operation, Terraform will download all the child modules from their
+        sources and place them in the appropriate relative directories.</para>
+
+        <para>Once again, the order is important. Initialize the modules in
+        the same order of precedence, virtual network first, the storage
+        account second, and then the AKS, or root.</para>
+
+        <para>To Initialize the Modules </para>
+
+        <orderedlist numeration="arabic">
+          <listitem>
+            <para>Change directory to the modules/virtual_network
+            directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Run terraform init in that directory:</para>
+
+            <programlisting>terraform init</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Confirm the module has been successfully initialized.</para>
+          </listitem>
+
+          <listitem>
+            <para>Change directory to the modules/storage_account
+            directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Run terraform init in that directory:</para>
+
+            <programlisting>terraform init</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Confirm the module has been successfully initialized.</para>
+          </listitem>
+
+          <listitem>
+            <para>Change directory to the root,
+            terraform-azurerm-hpcc directory.</para>
+          </listitem>
+
+          <listitem>
+            <para>Run terraform init in that directory:</para>
+
+            <programlisting>terraform init</programlisting>
+          </listitem>
+
+          <listitem>
+            <para>Confirm the module has been successfully initialized.</para>
+          </listitem>
+        </orderedlist>
+      </sect3>
+
+      <sect3 id="ApplyingTheTerraformModules">
+        <title>Applying the Terraform Modules</title>
+
+        <para>This step generates a Terraform Plan to confirm your
+        configuration choices. A Terraform plan is where after having
```

Comment:
A Terraform plan displays exactly what it is going to do so you can review it before applying it.